lib/services/suite/suite.go

```diff
@@ -606,10 +607,7 @@ func (s *ServicesTestSuite) TokenCRUD(t *testing.T) {
 	require.Equal(t, token.GetRoles().Include(types.RoleAuth), true)
 	require.Equal(t, token.GetRoles().Include(types.RoleNode), true)
 	require.Equal(t, token.GetRoles().Include(types.RoleProxy), false)
-	diff := s.Clock.Now().UTC().Add(defaults.ProvisioningTokenTTL).Second() - token.Expiry().Second()
-	if diff > 1 {
-		t.Fatalf("expected diff to be within one second, got %v instead", diff)
-	}
+	require.Equal(t, time.Time{}, token.Expiry())
```

Comment:
this check was always buggy, and the provision service does not set a default token expiry anymore since expiryless tokens are not allowed, so it's expected to return the zero time here